webpack-config-browser/jsdom-browser.js

```diff
@@ -0,0 +1,5 @@
+module.exports.JSDOM = class {
```

Comment:
making you code coupled to JSDOM and then remapping it to the browser fields is exactly the reversed way of how it should be done.